tests/modules/programs/senpai/example-settings-expected.conf

```diff
@@ -0,0 +1,15 @@
+address irc.libera.chat
+channel #rahxephon
+colors {
+  prompt 2
+}
+
+highlight guest senpai lenon
+nickname Guest123456
+pane-widths {
+  nicknames 16
+}
+
+password-cmd gopass show irc/guest
```

Comment:
Shouldn't this be `password-cmd "gopass show irc/guest"`?

Reply:
No, SCFG doesn't quote its strings. See: https://git.sr.ht/~taiite/senpai/tree/master/item/doc/senpai.5.scd#L199

Reply:
Ah, I understand. Thank you for the well reasoned response -- an option wrapping `execve` as a list of strings is much more correct :slightly_smiling_face: